peer_mentoring/account_management/templates/account_management/home.html

```diff
@@ -16,7 +16,9 @@ <h2>Man+</h2>
 {% if request.user.is_authenticated %}
   Welcome back {{request.user.username}} <a href="{% url 'account_logout' %}">logout</a>
 {% else %}
-  splash page here
+
+  {% include "account_management/splash.html" %}
```

Comment:
ripped a whole bunch of html out of base because it showed on the login page while fixing this issue